src/fungible-token.ts

```diff
@@ -123,4 +123,59 @@ export class FungibleToken {
     const nodeProvider = web3.getCurrentNodeProvider()
     return BigInt((await nodeProvider.fetchFungibleTokenMetaData(tokenId)).decimals)
   }
+
+  async airdropTx(
+    signer: SignerProvider,
+    valueOne: TokenAmount,
+    tokenOneId: string,
+    valueTwo: TokenAmount,
+    tokenTwoId: string,
+    valueThree: TokenAmount,
+    tokenThreeId: string
+  ) {
+    const tokenOneAddress = addressFromContractId(tokenOneId)
+    const tokenTwoAddress = addressFromContractId(tokenTwoId)
+    const tokenThreeAddress = addressFromContractId(tokenThreeId)
+    const tokenOneFaucet = TokenFaucet.at(tokenOneAddress)
+    const tokenTwoFaucet = TokenFaucet.at(tokenTwoAddress)
+    const tokenThreeFaucet = TokenFaucet.at(tokenThreeAddress)
+    const builder = TransactionBuilder.from(web3.getCurrentNodeProvider())
+    const bytecode = Airdrop.script.buildByteCodeToDeploy({
+      tokenOne: tokenOneFaucet.contractId,
```

Comment:
There's no need to have tokenOneFaucet etc, you can just pass tokenOneId. In `mintTx` it is that way for historical reasons (I forgot to change it...), because the entrypoint used to take contractAddress instead of id. You can change it there too.